virttest/qemu_vm.py

```diff
@@ -223,6 +223,35 @@ def __init__(self, name, params, root_dir, address_cache, state=None):
         self.last_boot_index = 0
         self.last_driver_index = 0
 
+    def get_command_style(self):
+        """
+        get data from file or input form parameter and then convert data to dict
+        :return: style data in dict
+        """
+        def __path(line):
+            res = {}
+            if line:
+                if not os.path.exists(line):
+                    abs_path = data_dir.get_data_dir() + "/" + line
+                with open(abs_path, "r") as f:
+                    res = json.load(f)
+            return res
+
+        def __json(line):
+            res = {}
+            if line:
+                res = json.loads(line)
+            return res
+
+        command_style_param = self.params.get("qemu_command_style")
+        flag, cont = command_style_param.split(":", 1)
+        sw = {"path": __path,
+              "json": __json}
+
+        if flag not in sw:
+            LOG.error("The format is NOT support! ")
```

Comment:
Should we also consider to either:
* fallback to use raw format?

or
* raise an `ValueError` accordingly?